### PR TITLE
[Issue-328]  Data load failing when the measure data is null

### PR DESCRIPTION
**Pull request by foryou2030**
fixed issue #328 
1 change calculateMaxMinUnique method name to calculateMaxMin 
2 null value don't need to call calculateMaxMin. fixed the load exception.

what's more, null measure value was stored as 0 , maybe null value should be stored as unique value.

### CODE REVIEW

**Comment by ravipesala** [comment]
Can one of the admins verify this patch?

File: integration/spark/src/test/scala/org/carbondata/integration/spark/testsuite/dataload/TestLoadDataWithNullMeasures.scala [hunk 1]
```diff
@@ -0,0 +1,25 @@
+package org.carbondata.integration.spark.testsuite.dataload
+
+import org.apache.spark.sql.common.util.CarbonHiveContext._
+import org.apache.spark.sql.common.util.QueryTest
+import org.scalatest.BeforeAndAfterAll
+
+/**
+ * Created by Administrator on 2016/5/17 0017.
```
Comment (eason-lyx) [inline integration/spark/src/test/scala/org/carbondata/integration/spark/testsuite/dataload/TestLoadDataWithNullMeasures.scala]: can u remove this?

**Comment by kumarvishal09** [comment]
retest this please

**Comment by foryou2030** [comment]
i have retest the newest code of master, but it doesn't working, the same error:

```
16/05/17 21:17:05 ERROR step.CarbonSliceMergerStep: [t3: Graph - Carbon Slice Mergert3][partitionID:t3] 
org.pentaho.di.core.exception.KettleException: 
org.carbondata.processing.merger.exeception.SliceMergerException: Problem while renaming inprogress folder to actual
Problem while renaming inprogress folder to actual

	at org.carbondata.processing.merger.step.CarbonSliceMergerStep.renameFolders(CarbonSliceMergerStep.java:168)
	at org.carbondata.processing.merger.step.CarbonSliceMergerStep.processRow(CarbonSliceMergerStep.java:118)
	at org.pentaho.di.trans.step.RunThread.run(RunThread.java:40)
	at java.lang.Thread.run(Thread.java:722)
Caused by: org.carbondata.processing.merger.exeception.SliceMergerException: Problem while renaming inprogress folder to actual
	at org.carbondata.processing.merger.step.CarbonSliceMergerStep.renameLoadFolderFromInProgressToNormal(CarbonSliceMergerStep.java:232)
	at org.carbondata.processing.merger.step.CarbonSliceMergerStep.renameFolders(CarbonSliceMergerStep.java:161)
	... 3 more
```

**Comment by kumarvishal09** [comment]
can u fix the issue, then we can merge

**Comment by foryou2030** [comment]
in this pr， i have fixed the load-failure issue, maybe u can merge it first.

**Comment by foryou2030** [comment]
retest this please

File: integration/spark/src/test/scala/org/carbondata/integration/spark/testsuite/dataload/TestLoadDataWithNullMeasures.scala [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+/**
+ * Test Class for data loading when there are null measures in data
+ *
+ */
+class TestLoadDataWithNullMeasures extends QueryTest with BeforeAndAfterAll {
+  override def beforeAll {
+    sql("CREATE CUBE carboncube DIMENSIONS (empno Integer, empname String, designation String, doj String, workgroupcategory Integer, workgroupcategoryname String, deptno Integer, deptname String, projectcode Integer, projectjoindate String, projectenddate String) MEASURES (attendance Integer,utilization Integer,salary Integer) OPTIONS (PARTITIONER [PARTITION_COUNT=1])")
+  }
+  test("test carbon table data loading when there are null measures in data") {
+    try {
```
Comment (gvramana) [inline integration/spark/src/test/scala/org/carbondata/integration/spark/testsuite/dataload/TestLoadDataWithNullMeasures.scala]: Please add a testcase for column with all nulls

**Comment by gvramana** [comment]
retest this please

**Comment by gvramana** [comment]
retest this please

**Comment by gvramana** [comment]
retest this please

**Comment by sraghunandan** [comment]
retest this please

**Comment by ravipesala** [comment]
add to whitelist

**Comment by ravipesala** [comment]
retest this please

**Comment by ravipesala** [comment]
add to whitelist

**Pull request merged**